source/_components/mythicbeastsdns.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Mythic Beasts DNS"
+description: "Keep your mythic beasts DNS updated"
+date: 2018-11-09 14:57
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Network
+featured: false
+ha_release: 0.82
+---
```

Comment:
Please add a logo and `ha_iot_class`.

Reply:
In his defence, none of the other similar components that do DNS updating, have a `ha_iot_class` defined. At this point, I'm kinda unsure myself on what the `ha_iot_class` should be?

Any suggestion on this @MartinHjelmare?

Reply:
`Cloud Push`?
it does push info out to the cloud.